cartesian/pfss_cartesian_obj.py

```diff
@@ -0,0 +1,133 @@
+import warnings
+
+import numpy as np
+import sunpy.map
+from astropy.io import fits
+import utils
+
+class Input:
+    r"""
+    Input to PFSS modelling.
+
+    This code uses a Cartesian geometry (x,y,z) for performing the extrapolation. 
+    Thus, make sure the grid defined is cartesian, and that solutions obtained are 
+    valid at all. 
+    Parameters
+    ----------
+    bz : sunpy.map.GenericMap
+        Boundary condition of LOS magnetic field at the inner surface.
+
+    Z_max : float
```

Comment:
Since the PFSS equations are length-scale free, this should be the number of grid cells in the Z direction instead.

Reply:
The code performs all computation in real units, as is the implementation in Nakagawa and Raadu. Furthermore, the resolution in x and y can be different. In such a case, I think it would be best to take the $Z_{max}$ and $dz$ in the same units as x and y, and not get into conversions from real to code units and back. What do you think?